packages/server/src/threads/automation.ts

```diff
@@ -70,6 +72,9 @@ class Orchestrator {
   _emitter: any
   _context: AutomationContext
   _job: Job
+  _loopStepOutputs: LoopStep[]
+  _stopped: boolean
+  _loopStep?: LoopStep
```

Comment:
Could all these `_whatever` be marked as `private`? I guess this is what we are trying to make with the `_`

Reply:
Sorted this!